jetty-io/src/main/java/org/eclipse/jetty/io/SocketChannelEndPoint.java

```diff
@@ -18,24 +18,15 @@
 
 package org.eclipse.jetty.io;
 
-import java.io.IOException;
-import java.net.InetSocketAddress;
 import java.net.Socket;
 import java.nio.channels.SelectableChannel;
 import java.nio.channels.SelectionKey;
 import java.nio.channels.SocketChannel;
 
-import org.eclipse.jetty.util.log.Log;
-import org.eclipse.jetty.util.log.Logger;
 import org.eclipse.jetty.util.thread.Scheduler;
 
 public class SocketChannelEndPoint extends ChannelEndPoint
```

Comment:
This class does very little more than `ChannelEndPoint` Shouldn't the two be merged?

Reply:
`ChannelEndPoint` becomes `SocketChannelEndPoint`, and `UnixEndPoint` extends `SocketChannelEndPoint`?
We may want to do this in 10, not in 9.4.